### ISSUE TITLE
Implement the nqp::print node

### ISSUE THREAD

**Issue by pmurias** [perl6/nqp#445]
The nqp::say node (NQPSayNode class) can be used as inspiration

**Comment by W4anD0eR96** [comment]
Sorry but where is the NQPSayNode?

**Comment by pmurias** [comment]
The truffle backend (which I'll get back to work on soonish) is in the
truffle branch .

**Comment by W4anD0eR96** [comment]
@pmurias sure I did it at ee3df8e2de0668a8b06fb19f216a9d9e076efe35

I noticed GraalVM some days ago, and will be glad to help push this branch.

However, I do not understand how truffle framework works. You see, I can imitate your implementation, but don't know things underneath. I will appreciate it if you give a brief talk about how truffle works. And as I understand, we convert source code to QAST then JAST then bc in nqp-j, how we tweak
